### PR TITLE
fix URL formatting, add note on **api**.github.com certificates

### PR DESCRIPTION
**Pull request by hf-kklein**

### CODE REVIEW

File: docs/guide-ssl-setup.md [hunk 1]
```diff
@@ -36,15 +36,18 @@ order: 10
...
 
 The certificate list should end up looking something like this:
 ![](img/strust.png)
 
+### Note on private GitHub repositories
+Private Repositories are supported but the login process requires the certificates for [https://**api**.github.com](https://api.github.com), too. Repeat the download and STRUST import steps as described in the sections above accordingly. (See also [Issue#1491](https://github.com/larshp/abapGit/issues/1491))
+
 ### Note about GitHub
-On [Feburary 22nd](https://githubengineering.com/crypto-removal-notice/), GitHub deprectated weak TSL connections.
+On [2018-02-22](https://githubengineering.com/crypto-removal-notice/), GitHub deprectated weak TSL connections.
```
Comment (sbcgua) [inline docs/guide-ssl-setup.md]: TLS ? (typo)

Comment (hf-kklein) [reply docs/guide-ssl-setup.md]: just fixed the typo in another commit.

File: docs/guide-ssl-setup.md [hunk 2]
```diff
@@ -36,15 +36,18 @@ order: 10
...
-* http://scn.sap.com/people/thomas.jung/blog/2004/11/17/bsp-a-developers-journal-part-xiv--consuming-webservices-with-abap
+Also see the following blog posts on SCN:
+* [Calling WebServices from ABAP via HTTPS/SSL with pfx certificates](http://scn.sap.com/people/jens.gleichmann/blog/2008/10/31/calling-webservices-from-abap-via-httpsssl-with-pfx-certificates)
+* [BSP a Developer’s Journal Part XIV – Consuming WebServices with ABAP](http://scn.sap.com/people/thomas.jung/blog/2004/11/17/bsp-a-developers-journal-part-xiv--consuming-webservices-with-abap)
 
 The certificate list should end up looking something like this:
 ![](img/strust.png)
 
+### Note on private GitHub repositories
+Private Repositories are supported but the login process requires the certificates for [https://**api**.github.com](https://api.github.com), too. Repeat the download and STRUST import steps as described in the sections above accordingly. (See also [Issue#1491](https://github.com/larshp/abapGit/issues/1491))
```
Comment (flaiker) [inline docs/guide-ssl-setup.md]: Just a small note:
The reason for the use of the api-certificate is not the repository being private but the operation requiring authentication. On public repositories you also need the certificate if you do any non-anonymous action like pushing commits or creating a branch.
The technical reason is that only api.github.com can supply the information whether two factor authentication is required or not. See #536

**Comment by flaiker** [comment]
I thought we already documented the need for the api certificate but apparently it's only _in the code_ so I think it makes sense to display it more prominently in the installation guidelines :+1:
#1419
https://github.com/larshp/abapGit/blob/d19ce7c4e3ed714af0c2cb0d1be3020d4650dc29/src/http/zcl_abapgit_2fa_github_auth.clas.abap#L338-L343

File: docs/guide-ssl-setup.md (hunk 2 above)
Comment (hf-kklein) [reply docs/guide-ssl-setup.md]: I added a new commit that summarizes your comment.

**Comment by hf-kklein** [comment]
> I thought we already documented the need for the api certificate

Well, the SSL Test Report actually returns an error in this case but since I already installed a package using AbapGit I thought this was just fine 😁

**Comment by larshp** [comment]
thanks, welcome to abapGit as a contributor 🎉

**Pull request merged**